adoc/deployment-aws.adoc

```diff
@@ -93,6 +93,176 @@ requires your credentials. These can be obtained by following these steps:
 * Click menu:Create Access Key[] on the "Security Credentials" tab.
 * Note down the newly created _Access_ and _Secret_ keys.
 
+=== Create IAM Role, Role Policy, and Instance Profile through AWS CLI ===
+
+For user who does not have permission to create IAM role, role policy, and instance profile using terraform, devops should create them for you using instruction below:
+
+* `STACK_NAME`: Cluster Stack Name
+
+Install AWS CLI:
+
+----
+curl "https://d1vvhvl2y92vvt.cloudfront.net/awscli-exe-linux-x86_64.zip" -o ./awscliv2.zip
+unzip awscliv2.zip
+sudo ./aws/install
```

Comment:
Follow the AWS doc https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2-linux.html
```
curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip"
unzip awscliv2.zip
sudo ./aws/install
```

Besides, seems like we should consider packaging `awscli` https://build.opensuse.org/package/show/home:dirkmueller:branches:Cloud:Tools/awscli

Reply:
It seems like AWS have updated the document, since my script is from a while ago.  We do have it in `https://build.suse.de/package/show/SUSE:SLE-15-SP1:Update/aws-cli`, but version is 1.16.223. From AWS, the version is 2.0.5.

Reply:
functin work with package aws-cli.. upated the cli to use zyper.